.eslintrc.yml

```diff
@@ -1 +1,10 @@
-extends: 'metarhia'
+extends:
+  - metarhia
+  - plugin:prettier/recommended
+  - prettier/react
```

Comment:
You don't need any of these except `metarhia` and `plugin:prettier/recommended`.